config/config.go

```diff
@@ -171,6 +171,7 @@ type IstioConfig struct {
 	EnvoyAdminLocalPort      int               `yaml:"envoy_admin_local_port,omitempty"`
 	IstioIdentityDomain      string            `yaml:"istio_identity_domain,omitempty"`
 	IstioInjectionAnnotation string            `yaml:"istio_injection_annotation,omitempty"`
+	IstioInjectorName        string            `yaml:"istio_injector_name,omitempty"`
```

Comment:
I think this should be called "IstioInjectorConfigMapName" (istio_injector_config_map_name) since that is what it is. Just saying "injector name" is confusing because the name gives no indication it is a config map name. Clarity is a good thing, even if it increases verbosity slightly.

(side note: we don't have to do this now, but "ConfigMapName" should probably be called IstiodConfigMapName to be more clear. But again no need to do that now because that will affect user-facing Kiali CRs).